test/cmdlineargs.jl

```diff
@@ -69,6 +69,7 @@ let exename = `$(Base.julia_cmd()) --precompiled=yes`
     end
 
     # --procs
+    # FIXME: is the -p 2 option incompatible with --startup-file=no ?
     @test readchomp(`$exename -q -p 2 -e "println(nworkers())"`) == "2"
```

Comment:
Here's a problem:

```❯ ./julia -q --startup-file=no -e "println(nworkers())"
1

rrock at Uluru in ~/Documents/Code/julia5  rsr/test-ignore-startup
❯ ./julia -q -p 2 --startup-file=no -e "println(nworkers())"
ERROR: LoadError: Don't include me!
 in error(::String) at ./error.jl:21
 in include_from_node1(::String) at ./loading.jl:426
 in try_include at ./client.jl:200 [inlined]
 in load_juliarc() at ./client.jl:278
 in process_options(::Base.JLOptions) at ./client.jl:216
 in _start() at ./client.jl:318
while loading /Users/rrock/.juliarc.jl, in expression starting on line 1
ERROR: LoadError: Don't include me!
 in error(::String) at ./error.jl:21
 in include_from_node1(::String) at ./loading.jl:426
 in try_include at ./client.jl:200 [inlined]
 in load_juliarc() at ./client.jl:278
 in process_options(::Base.JLOptions) at ./client.jl:216
 in _start() at ./client.jl:318
while loading /Users/rrock/.juliarc.jl, in expression starting on line 1

[hangs here]
```

It looks like the `-p 2` option ignores the `--startup-file=no` option.  Should I file a separate issue for this?